r/README.md

```diff
@@ -75,11 +75,11 @@ and thus are only available in R >= 4.0.
 ### Installing a development version
 
 Development versions of the package (binary and source) are built
-nightly and hosted at <https://arrow-r-nightly.s3.amazonaws.com>. To
+nightly and hosted at <https://nightlies.apache.org/arrow/r>. To
```

Comment:
https://nightlies.apache.org/arrow/r does a 301 redirect to https://nightlies.apache.org/arrow/r/ so everywhere there is a URL that CRAN may check, we need the trailing slash.

```suggestion
nightly and hosted at <https://nightlies.apache.org/arrow/r/>. To
```

Reply:
oh good catch!